src/script/interpreter.cpp

```diff
@@ -1584,8 +1639,10 @@ bool SignatureHashSchnorr(uint256& hash_out, const ScriptExecutionData& execdata
 
     hash_out = ss.GetSHA256();
     return true;
+
 }
 
+
```

Comment:
in commit 51f0bdd36328f1c9334ad455a56f7cb47d76abda

nit unneeded spacing

Reply:
i think 1645 is needed, 1642 could probaby go

Reply:
removing both; not sure why they were there to begin with.